### PR TITLE
Further PL translation fixes

### PR DESCRIPTION
**Pull request by gienkov**
Used tabler this time and this is why there are so many changes. Good tool I must say!

### CODE REVIEW

File: addons/missileguidance/stringtable.xml [hunk 1]
```diff
@@ -28,80 +28,8 @@
...
-          <Czech>Hellfire II AGM-114K laserem naváděná střela</Czech>
-          <Italian></Italian>
-          <Portuguese></Portuguese>
-          <Hungarian></Hungarian>
-          <Russian></Russian>
-        </Key>
-    
-  </Package>
+        <Key ID="STR_ACE_Hellfire_AGM114K"><English>Hellfire II AGM-114K Missile</English><Spanish></Spanish><French></French><Polish>Hellfire II AGM-114K</Polish><German></German><Czech>Hellfire II AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_Short"><English>AGM-114K</English><Spanish></Spanish><French></French><Polish>AGM-114K</Polish><German></German><Czech>AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_desc"><English>Hellfire II AGM-114K Laser Guided Missile</English><Spanish></Spanish><French></French><Polish>Laserowo naprowadzana rakieta Hellfire II AGM-114K</Polish><German></German><Czech>Hellfire II AGM-114K laserem naváděná střela</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key></Package>
```
Comment (esteldunedain) [inline addons/missileguidance/stringtable.xml]: @bux578, could tabler be failing here?

Comment (bux578) [inline addons/missileguidance/stringtable.xml]: He's using the latest public version that uses 4 spaces for serializing and `<Package> ` gets ignored.

Comment (esteldunedain) [inline addons/missileguidance/stringtable.xml]: I mean that the line breaks are missing

Comment (gienkov) [inline addons/missileguidance/stringtable.xml]: Didn't saw that one, fixing

Comment (bux578) [inline addons/missileguidance/stringtable.xml]: That's probably because of these invalid comments: `/* Weapons */`

**Comment by gienkov** [comment]
Fixed that one

**Comment by gienkov** [comment]
Ready

**Comment by bux578** [comment]
:+1:

**Pull request merged**